examples/Azure_IoT_Adu_ESP32/SampleAduJWS.h

```diff
@@ -37,11 +37,24 @@
 
 namespace SampleJWS
 {
+
+/**
+ * @brief Holds the values of the root key used to verify the JWS signature.
+ */
+typedef struct RootKey
+{
+    az_span root_key_id;
+    az_span root_key_n;
+    az_span root_key_exponent;
+} RootKey;
```

Comment:
There might be a chance of name conflicts because it's quite a generic name.
Suggestion: at least JwsRootKey

Reply:
If it's namespaced wouldn't that be okay?

Reply:
For Arduino yes, since it's c++. But if this header is taken out for some customer project that is pure c, namespaces would not apply. On the other hand, these namespaces are not declared using an ifdef guard to check if it's a c++ compilation, so it would fail anyway.
So in this case, fine. We will change this if any customer complains.